### PR TITLE
add cubic bezier example

### PR DESCRIPTION
**Pull request by mzdraper**
Interpolate expression using cubic bezier curve to style bathymetry data example because there aren't any current examples using this expression.

### CODE REVIEW

File: docs/pages/example/style-ocean-depth-data.html [hunk 1]
```diff
@@ -0,0 +1,64 @@
+<div id='map'></div>
+
+<script>
+var map = new mapboxgl.Map({
+    container: 'map',
+    style: 'mapbox://styles/mapbox/streets-v10',
+    center: [142.591669, 11.373335], // coordinate position
+    zoom: 3, // starting zoom
+    minZoom: 3, // minimum zoom
+    maxZoom: 5, // maximum zoom
```
Comment (ryanhamley) [inline docs/pages/example/style-ocean-depth-data.html]: I don't think we need these comments on lines 7-10 and lines 15 and 21. But can you add one or two explanatory comments that briefly explain how the expression is working?

**Comment by ryanhamley** [comment]
This is a super cool example!

**Comment by malwoodsantoro** [comment]
So cool @mzdraper 👍 Makes sense @mollymerp. Maybe we could refactor this to use `step` since it's just a cool example and create something separate to demonstrate cubic bezier. Any ideas of a visualization that would better showcase cubic bezier?

**Comment by mollymerp** [comment]
agree that it is a super cool example! 👍 to either converting this to `step` as is or using a cubic bezier function with fewer stops – the following looks pretty good to me 😊 
```js
"fill-color": [ "interpolate",
                [ "cubic-bezier",
                    0, 0.5,
                    1, 0.5 ],
                ["get", "DEPTH"],
                200,  "#78bced",
                9000, "#15659f"
              ]
```

File: docs/pages/example/style-ocean-depth-data.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        "id": "10m-bathymetry-81bsvj",
+        "type": "fill",
+        "source": "10m-bathymetry-81bsvj",
+        "source-layer": "10m-bathymetry-81bsvj",
+        "layout": {},
+        "paint": {
+            "fill-outline-color": "hsla(337, 82%, 62%, 0)",
+            // cubic bezier is a four point curve for smooth and precise styling
+            // adjust the points to change the rate and intensity of interpolation
+            "fill-color": "fill-color": [ "interpolate",
```
Comment (mollymerp) [inline docs/pages/example/style-ocean-depth-data.html]: copy pasta 🍝 here!

Comment (mzdraper) [reply docs/pages/example/style-ocean-depth-data.html]: ops, forgot to commit that change - thanks!

**Pull request merged**